app/views/includes/foot.nunjucks

```diff
@@ -0,0 +1,4 @@
+<script src="{{ SITE_ROOT }}/js/vendor/jquery-1.12.4.min.js"></script>
+<script src="{{ SITE_ROOT }}/js/app.js"></script>
```

Comment:
This file doesn't exist atm and is causing 404s on the front end, can it be removed please?